components/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/util/SCIMProviderConstants.java

```diff
@@ -52,6 +52,8 @@
     public static final String DISPLAY_NAME = "displayName";
     public static final String WILDCARD_ASTERISK = "*";
 
+    public static final String ELEMENT_NAME_SHOW_ALL_DETAILS = "ShowAllDetails";
```

Comment:
Shall we change this variable name? Also shall we change the value or tag name as showAllUserDetails or something like that?